Gemfile

```diff
@@ -44,30 +45,28 @@ gem 'fog'
 gem 'thin'
 gem 'roadie-rails', '~> 1.1.0'
 gem 'sequenced', '~> 2.0.0'
-gem 'closure_tree', '~> 5.2.0'
+# gem 'closure_tree', '~> 5.2.0'
 gem 'bing_translator'
 gem 'http_accept_language'
 gem 'intercom'
-gem 'intercom-rails'
-gem 'underscore-rails'
-gem "font-awesome-sass",  '~> 4.3.2'
+# gem 'intercom-rails'
+# gem 'underscore-rails'
+# gem "font-awesome-sass",  '~> 4.3.2'
 gem 'coffee-rails'
 gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'uglifier'
 gem 'bootstrap-sass', '~> 3.3.4'
 gem 'sass-rails', '>= 3.2'
-gem 'autoprefixer-rails'
-gem 'modernizr-rails'
-gem 'jquery-fileupload-rails'
-gem 'momentjs-rails'
+# gem 'autoprefixer-rails'
+# gem 'modernizr-rails'
+# gem 'jquery-fileupload-rails'
+# gem 'momentjs-rails'
 gem 'ruby-progressbar'
-gem 'responders', '~> 2.0'
+# gem 'responders', '~> 2.0'
 gem 'activerecord_any_of'
 gem 'rack-attack', '~> 4.2.0'
-gem 'ahoy_matey'
+gem 'ahoy_matey', '~> 1.1.1'
 gem 'ahoy_email'
-gem 'oj'
```

Comment:
Oh cool, I'll throw it back in.

On Tue, Apr 12, 2016 at 9:55 AM, Robert Guthrie <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/loomio/loomio/pull/3155#discussion_r59288745>:
>
> >  gem 'ahoy_email'
> > -gem 'oj'
>
> and it may affect json serialization in angular
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3155/files/48997303b45e67b490effb06c8d414b9c93be7f7#r59288745>
>